.github/workflows/daily-build.yml

```
@@ -0,0 +1,277 @@
+        shell: cmd
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew.bat build
+
+  # ─────────────────────────────────────────────────────────────────────────
+  # Ballerina VSCode extension — build VSIX + tests
+  # ─────────────────────────────────────────────────────────────────────────
+  Build:
+    name: Extension build
+    uses: ./.github/workflows/build.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/build.yml": open /home/jailuser/git/.github/workflows/build.yml: no such file or directory

(workflow-call)